src/cli/self_update.rs

```diff
@@ -19,8 +18,12 @@ use std::str::FromStr;
 #[derive(Debug, clap::Parser)]
 pub struct Args {
     /// The desired version (to downgrade or upgrade to).
-    #[clap(long)]
+    #[clap(long, conflicts_with = "force_latest")]
     version: Option<Version>,
+
+    /// Force upgrade to the latest version, ignore with the current version.
+    #[clap(long, default_value_t = false, conflicts_with = "version")]
```

Comment:
I think a general `--force` would be more applicable, to avoid any of the warning and just force the version through.